src/index.tsx

```diff
@@ -59,7 +61,9 @@ if (!(window as any).Cypress) {
 ReactDOM.render(
   <Provider store={store}>
     <ConnectedRouter history={history}>
-      <App />
+      <GAListener trackingId={process.env.NODE_ENV === "production" ? "UA-775587-9" : undefined}>
```

Comment:
Codacy found an issue: [Unexpected use of undefined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6088517)